docs/high-availability/srv_kubernetes/apiserver.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
These are the same files as docs/high-availability/apiserver.crt and docs/high-availability/apiserver.key which are added in this PR